paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenUI.kt

```diff
@@ -0,0 +1,41 @@
+package com.stripe.android.paymentsheet.verticalmode
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.unit.dp
+import com.stripe.android.paymentsheet.DisplayableSavedPaymentMethod
+import com.stripe.android.paymentsheet.R
+
+@Composable
+internal fun ManageScreenUI(interactor: ManageScreenInteractor) {
+    val horizontalPadding = dimensionResource(
+        id = R.dimen.stripe_paymentsheet_outer_spacing_horizontal
+    )
+
+    val state by interactor.state.collectAsState()
+
+    Column(
+        modifier = Modifier.padding(horizontal = horizontalPadding),
+        verticalArrangement = Arrangement.spacedBy(12.dp)
+    ) {
+        state.paymentMethods?.forEach {
```

Comment:
should we default paymentMethod to an empty list?

Another idea is we should update the interactor to take in specific things that we know can't be null/empty, otherwise we'd never get here.

-- Just an idea, non blocking.